components/org.wso2.carbon.status.dashboard.core/src/main/java/org/wso2/carbon/status/dashboard/core/internal/services/PermissionGrantServiceComponent.java

```diff
@@ -64,10 +64,10 @@ protected void start(BundleContext bundleContext) {
             logger.info("Status dashboard permission grant service component is activated.");
         }
         try {
-            DeploymentConfigs resolvedConfiguration=DashboardDataHolder.getInstance()
+            DeploymentConfigs resolvedConfiguration=MonitoringDataHolder.getInstance()
```

Comment:
space after and before =

Reply:
Fixed in ea04a21de7eed189e8627127fcd8e4c9c600b999